docs/about/release-notes.rst

```diff
@@ -22,6 +22,7 @@ Bugfixes
 * Fix :func:`scipp.isclose` and :func:`scipp.allclose` to support arguments without a unit `#2528 <https://github.com/scipp/scipp/pull/2528>`_.
 * :func:`scipp.to_unit` avoids a rounding problem when converting datetimes. This previously led to errors, e.g., of about 300 nanoseconds when converting a current (2020s) datetime64 from seconds to nanoseconds `#2533 <https://github.com/scipp/scipp/pull/2533>`_.
 * Fix handling of keyword arguments in :func:`scipp.optimize.curve_fit`. They were previously checked for conflicts but otherwise ignored `#2545 <https://github.com/scipp/scipp/pull/2545>`_.
+* Fix segmentation fault in slic-by-list. Found on an Apple arm64 when compiled with Apple Clang, but probably affected other systems as well `#2548 <https://github.com/scipp/scipp/pull/2548>`_.
```

Comment:
No need for release notes, I think you introduced this in #2538, i.e., after the last release?

Reply:
Oh, you are right. I thought this was already the case before. I guess that explains why it didn't show up with apple clang in the past.